.dprint.json

```diff
@@ -9,8 +9,12 @@
   "json": {
     "deno": true
   },
+  "exec": {
+    "associations": "**/*.rs",
+    "rustfmt": "rustfmt"
```

Comment:
The exec plugin has no knowledge of `rustfmt`. This makes it calls the `rustfmt` cli and format over stdin.

https://github.com/dprint/dprint-plugin-exec#configuration